SoftwareCo/SoftwareCo/Managers/DocEventManager.cs

```diff
@@ -180,7 +170,8 @@ public async void DocEventsOnDocumentOpenedAsync(Document document)
             InitPluginDataIfNotExists();
             _pluginData.InitFileInfoIfNotExists(fileName);
 
-            /**
+            TrackerUtilManager.TrackEditorFileActionEvent("file", "open", fileName);
```

Comment:
you have a handle to the Document here, why not pass that to the Tracker itself?

Document has these properties:
https://docs.microsoft.com/en-us/dotnet/api/envdte.document?view=visualstudiosdk-2017#properties